packages/core/src/channels/assembly-channel.class.ts

```diff
@@ -0,0 +1,29 @@
+import { Subscription } from '../event-emitters';
+import { ProbeManager } from '../probes';
+import { AuguryEventAssembler } from './assemblers';
+import { ChannelManager } from './channel-manager.class';
+import { Channel } from './channel.class';
+
+export class AssemblyChannel<Output> extends Channel<Output> {
```

Comment:
The `Output` threw me off originally because I thought it was an external type and not a generic type because it's spelt out fully. It's typically more common to defaul to `<T>` for one arg and single letters for others whhen multiple. Perhaps using that convention would be more familiar and approachable.